src/Endpoints/OAuth2.php

```diff
@@ -35,7 +35,7 @@ public function getTokensByCode(string $clientId, string $clientSecret)
 
         $options['headers']['content-type'] = 'application/x-www-form-urlencoded';
 
-        $response = $this->client->request('post', $this->endpoint, $options);
+        $response = $this->client->request('post', $this->endpoint, $options, '', false);
```

Comment:
> I also don't understand why the linters are failing with my change. 😭

<img width="888" alt="CleanShot 2023-08-03 at 09 53 48@2x" src="https://github.com/ChromaticHQ/orange-dam-php/assets/20355/3fdf931e-688b-4646-8c24-353b9e608b06">


@adamzimmermann [RemoveExtraParametersRector](https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#removeextraparametersrector) It doesn't seem to be able to identify what these extra parameters are for.

I looked in the Guzzle codebase and I am not seeing them either:

<img width="836" alt="CleanShot 2023-08-03 at 10 02 59@2x" src="https://github.com/ChromaticHQ/orange-dam-php/assets/20355/34d00a6f-00d4-460c-a945-6e9f1605332e">


* [v6](https://github.com/guzzle/guzzle/blob/a52f0440530b54fa079ce76e8c5d196a42cad981/src/ClientInterface.php#L57)
* [v7](https://github.com/guzzle/guzzle/blob/cd634c213de11e0f229651173428f5c27d398628/src/Client.php#L185)

Reply:
I was about to say that we are using `Chromatic\OrangeDam\Http::Request` here which does have those parameters, but then I realized that isn't true like I thought it was yesterday.

But it _sure seems_ like that is what is happening based upon my changes solving the issue and the code working. 🤔 

Let me explore this a bit more, maybe we can pair on this too.

Reply:
![Screenshot 2023-08-03 at 3 16 39 PM](https://github.com/ChromaticHQ/orange-dam-php/assets/1349906/f90c76b0-1939-49eb-999e-c3e0031946a9)

1. The client is the custom one as I expected (depending on the day you go off of 😂 ), thus explaining why my "fix" works.
2. I checked out `v1.0.0-alpha.3` and everything worked just fine when I revert my change. So it's definitely something between `v1.0.0-alpha.3` and `1.0.0`.
3. Now to understand what changed. I'm struggling to spot it in the diff still. 

🕵🏼

Reply:
Well, this would explain it. The diff on this PR simply reverts a change made during the code linting clean-up. That explains the issue.

![Screenshot 2023-08-03 at 4 55 08 PM](https://github.com/ChromaticHQ/orange-dam-php/assets/1349906/549440a6-931a-4c92-b1ca-4a9c15b1508d)

Now to understand what the correct fix is or how to make linting happy.

[`v1.0.0-alpha.3...v1.0.0?w=1`#diff-e0d0141917](https://github.com/ChromaticHQ/orange-dam-php/compare/v1.0.0-alpha.3...v1.0.0?w=1#diff-e0d014191715cd85169e2db06606486f7e6e713a511556f6214aeba926378940R38)

Reply:
Tracked it down. A code comment was throwing the linter off.

- 3a466cb7fd97934db4cc5759a32940a44e3d1d8a

All is good now! 🎉

Reply:
That explains it! Nice!